src/gui_common/CustomRichTextLabel.cs

```diff
@@ -55,6 +57,8 @@ public enum ThriveBbCode
             // the compound names anyway
             extendedBbcode = value;
 
+            extendedBbcodeTranslated = TranslationServer.Translate(value);
```

Comment:
Can't this have problems regarding language change? This this should only be done in the ParseCustomTags method, I think. That's not super perfect either, but I think it'll be better than translating this here.